### PR TITLE
MGMT-12804: Make assisted-service-index:ocm-2.7 image available in Prow

### PR DESCRIPTION
**Pull request by eliorerz**
Manually create an ocm-2.7 release of the assisted-service-index from latest

/cc @osherdp 
/cc @adriengentil

### CODE REVIEW

**Review by osherdp — APPROVED** [review]

**Review by adriengentil — APPROVED** [review]

File: clusters/app.ci/multi-arch/namespaces.yaml [hunk 1]
```diff
@@ -4,11 +4,11 @@ items:
 - apiVersion: v1
   kind: Namespace
   metadata:
-    name: coreos-arm64
+    name: azure-arm64
```
Comment (osherdp) [inline clusters/app.ci/multi-arch/namespaces.yaml:7]: is it required?

**Review by osherdp — APPROVED** [review]

**Comment by bear-redhat** [comment]
Are the changes in `clusters/app.ci/multi-arch/namespaces.yaml` required?

File: clusters/app.ci/multi-arch/namespaces.yaml (hunk 1 above)
Comment (eliorerz) [reply clusters/app.ci/multi-arch/namespaces.yaml:7]: I got this error on `multi-arch-gen-valid` test
```
ERROR: To update the the multi-arch image mirroring run the following and
ERROR: commit the result:
ERROR: $ make multi-arch-gen
```

**Comment by mkowalski** [comment]
@eliorerz @bear-redhat as I read changes in this file, they are only cosmetic and change order of the objects, but not the number and content

File: clusters/app.ci/multi-arch/namespaces.yaml (hunk 1 above)
Comment (osherdp) [reply clusters/app.ci/multi-arch/namespaces.yaml:7]: Apparently it returns a different order in my env. I think ``ruamel.yaml`` has some regression which makes it returning stuff in different order sometimes
I recreated this change in https://github.com/openshift/release/pull/34728 without reordering

**Pull request closed without merging**